src/libs/ReportUtils.js

```diff
@@ -1256,7 +1253,7 @@ function getIOUReportActionMessage(type, total, comment, currency, paymentType =
             iouMessage = `deleted the ${amount} request${comment && ` for ${comment}`}`;
             break;
         case CONST.IOU.REPORT_ACTION_TYPE.PAY:
-            iouMessage = isSettlingUp ? `settled up ${amount}${paymentMethodMessage}` : `sent ${amount}${comment && ` for ${comment}`}${paymentMethodMessage}`;
+            iouMessage = isSettlingUp ? `paid ${amount}${paymentMethodMessage}` : `sent ${amount}${comment && ` for ${comment}`}${paymentMethodMessage}`;
```

Comment:
I know we do this for all the cases, but I just realized that the optimistic action will show in English even if we have the UI in Spanish, right? Or is this used for something else?

Reply:
Its coming from PHP so its not translated thats why we add it as english